docs/UserGuide.md

```diff
@@ -69,85 +69,34 @@ FoodWhere (FW) is a **desktop app for managing food reviews, optimized for use v
 
 Shows a message explaning how to access the help page.
 
-![help message](images/helpMessage.png)
+![help message](images/help.png)
 
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a stall: `sadd`
 
-Adds a person to the address book.
+Adds a stall.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
-
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
-</div>
+Format: `sadd n/NAME a/ADDRESS [d/DETAILS]...`
```

Comment:
Earlier, we used the `…` to denote multiple times - I think we should stick to the format that AB3 uses. What do you think?